### ISSUE TITLE
Central punctum opening in cyst

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10830]
**Preferred term label:**
Central punctum opening in cyst

**Synonyms**

**Definition (free text, please give PubMed ID)**
A small round opening in the top of the cyst that may look like a tiny dot or pit

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Epidermoid cyst

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I am not an expert, but it seems that Epidermoid cysts usually have this finding ("A central, dark comedone opening (punctum) is often described." see https://pubmed.ncbi.nlm.nih.gov/29763149/). Therefore, we should probably either just include it in the definition of make terms such as "Epidermoid cyst with/without central comedome opening".
Thoughts?

**Comment by MickeySegal** [comment]
We strive to have findings that average clinicians can recognize.  The definition at https://hpo.jax.org/browse/term/HP:0200040 is good, but few will see this and choose the finding.  This is why it seems reasonable to have a finding for the observation of a central punctum.  But looking at https://dermnetnz.org/topics/cutaneous-cysts-and-pseudocysts the central punctum is mentioned only for epidermoid cysts.

**Comment by pnrobinson** [comment]
@MickeySegal 
would it be a good idea to append the sentence "A central, dark comedone opening (punctum) may be present." to the current definition of Epidermoid cyst and add the PMID e https://pubmed.ncbi.nlm.nih.gov/29763149/).

**Comment by MickeySegal** [comment]
That is helpful information to add, but I'm thinking of the value of UMLS picking up HPO terms and feeding Natural Language Processing.  If a clinician observes a central punctum without ever having heard of an epidermoid cyst, creating this observational finding would allow that clinician to converge on the diagnosis without ever having heard of an epidermoid cyst.

Such things happen.  We quote one example on our web site: "“Using the resources provided I became convinced we should consider Morvan Syndrome in our differential diagnosis. The software suggested the relevant test. I ordered it and the test results were positive. I felt like I had superpowers to diagnose a disease I didn’t know beforehand.”"

**Comment by pnrobinson** [comment]
@MickeySegal  -- 
I think it is best to add this as a related synonym
Central punctum opening in cyst
This will allow the term to be found by text mining without adding a partially overlapping term (which has other disadvantages)!

**Comment by MickeySegal** [comment]
I added https://hpo.jax.org/browse/term/HP:0200040 to both our "Central punctum opening in cyst" and "Epidermoid cyst" findings, with each alluding to the other in our synonyms.
